BedrockServer.cpp

```diff
@@ -1807,16 +1859,28 @@ void BedrockServer::_prePollPlugins(fd_map& fdm) {
 }
 
 void BedrockServer::_postPollPlugins(fd_map& fdm, uint64_t nextActivity) {
+    // Only pass timeouts for transactions belonging to timed out commands.
+    uint64_t now = STimeNow();
+    map<SHTTPSManager::Transaction*, uint64_t> transactionTimeouts;
+    auto timeoutIt = _outstandingHTTPSCommands.begin();
+    while (timeoutIt != _outstandingHTTPSCommands.end() && (*timeoutIt)->timeout() < now) {
```

Comment:
Same answer, I explicitly added a sorting function for this here (since `std::less` doesn't do anything useful with `BedrockCommand*`s): https://github.com/Expensify/Bedrock/blob/3757ab830d6b08a4efb685789991f7f4efb1e661/BedrockServer.h#L405-L410

For the same reason - this way we only have to look at commands that have already timed out (since they're at the front of the list).